internal/entrypoint/routes.go

```diff
@@ -73,3 +73,9 @@ func registerTokenMissingRoutes(r *mux.Router, serverIP, serverPort, publicKeyPa
 	r.HandleFunc(path.Join("/token", oidcserver.TokenMissing), oidc.IssueToken).Methods(http.MethodPost)
 	r.HandleFunc(path.Join("/cert", oidcserver.TokenMissing), oidc.IssueCertificate).Methods(http.MethodGet)
 }
+
+func registerHealthzRoutes(r *mux.Router, serverIP, serverPort, publicKeyPath, privateKeyPath string, dev bool) {
+	oidc := oidcserver.NewOIDCServer(oidcserver.ServerFlowHealthz, serverIP, serverPort, publicKeyPath, privateKeyPath, dev)
```

Comment:
You don’t need to initialize a new oidc instance for the health endpoint. This can just be a function right ?

Reply:
I agree with @sibicramesh here, it should not be a new oidc instance. It can be a route on the existing instance

Reply:
I made that change here 06eff30a1e8e1ad35219cf1ebe91b7f3a4427219